prime-dotnet-webapi/Services/EmailDocumentsService.cs

```diff
@@ -20,12 +24,14 @@ public class EmailDocumentsService : BaseService, IEmailDocumentsService
         private readonly IOrganizationService _organizationService;
         private readonly IPdfService _pdfService;
         private readonly IRazorConverterService _razorConverterService;
+        private readonly IMapper _mapper;
 
         public EmailDocumentsService(
             ApiDbContext context,
-            ILogger<EmailDocumentsService> logger,
             IDocumentAccessTokenService documentAccessTokenService,
             IDocumentManagerClient documentClient,
+            ILogger<EmailDocumentsService> logger,
```

Comment:
While our DI is alphabetical, the dbContext and logger always appear first because they are needed by the base service.

Reply:
Updated